eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/LedgerSpec.hs

```diff
@@ -237,6 +238,9 @@ spec = do
           sendCoinTo addr amount
 
       inputsCommon <- replicateM 5 newInput
+      inputsCommonNES <- case NES.fromFoldable inputsCommon of
+        Nothing -> error "Impossible empty set"
+        Just nes -> pure nes
```

Comment:
Again, I didn't want to add an unsafe constructor for `NonEmptySet`, so we error out here. I'd be happy with this because it's in tests, but I wonder if the custom is different around the codebase 🙂